torchvision/ops/_utils.py

```diff
@@ -61,3 +61,64 @@ def split_normalization_params(
         else:
             other_params.extend(p for p in module.parameters() if p.requires_grad)
     return norm_params, other_params
+
+
+def set_weight_decay(
+    model: torch.nn.Module,
+    weight_decay: float,
+    norm_weight_decay: Optional[float] = None,
+    norm_classes: Optional[List[type]] = None,
+    bias_weight_decay: Optional[float] = None,
+    custom_keys_weight_decay: Optional[Dict[str, float]] = None,
+):
+    if not norm_classes:
```

Comment:
Given that you now provide a custom_keys solution, I don't think we need to support also custom `norm_classes`. We can just hardcode them and ask users to use the custom_keys feature if they need something different. Thoughts?

Reply:
yes, but the custom keys in this function are only applied in the user-defined `Parameters`, the predefined modules in torch (eg. nn.LayerNorm) are not considered.

Reply:
I think I must give some explanation：
- here we only user-defined `Parameters` for custom_keys
- the norm layer is firstly considered, if we set norm_weight_decay, the `bias parameter` in the norm layer use the norm_weight_decay even we set bias_norm_weight_decay
- we only consider the bias parameter in the predefined modules in torch.

Reply:
Thanks a lot for the clarifications, this was useful! 

The side-effect of the bias on Norm layers wan't apparent to me while reading the code. Could we update the code (refactoring + simplification + documenting) so that these corner cases are more apparent?

What I originally had in mind is that the functionality of Normalization doesn't work exactly as someone wants, they can easily overwrite its behaviour using the custom  keys. I think the custom keys should overwrite all other options. Let me know your thoughts. Happy to have a chat offline if you think that helps clarify the points easier.